app/models/ocelot/stanzas/RowGroup.scala

```diff
@@ -20,7 +20,10 @@ import models.ocelot.isLinkOnlyPhrase
 import models.ocelot.Phrase
 
 case class RowGroup (override val next: Seq[String], group: Seq[Row], stack: Boolean) extends VisualStanza with Populated {
-  lazy val paddedRows = group.map(row => (row.cells ++ Seq.fill(row.cells.length - row.cells.size)(Phrase())))
+  lazy val paddedRows = {
+    val maxRowLength = group.map(_.cells.length).max
+    group.map(row => (row.cells ++ Seq.fill(maxRowLength - row.cells.size)(Phrase())))
+  }
```

Comment:
Is the function isSummaryList safe? Do we know that the row length is definitely 3 before evaluating r.cells(2)?

Reply:
yes, because if we are executing r.cells(2), the previous or'ed r.cells.length < 3 must have failed